corehq/apps/locations/models.py

```diff
@@ -372,7 +372,10 @@ def __init__(self, *args, **kwargs):
             kwargs['lineage'] = lineage
             del kwargs['parent']
 
+        location_type = kwargs.get('location_type', None)
         super(Document, self).__init__(*args, **kwargs)
+        if location_type:
+            self.location_type = location_type
```

Comment:
This turns
```python
loc = Location(name="Boston", location_type="city")
```
into this:
```python
loc = Location(name="Boston")
loc.location_type = location_type
```
So that if you initialize a location like normal, it still goes through the `setter`.